game/services.py

```
@@ -15,27 +16,27 @@ def cleanup_stale_games():
-    
+
     deleted_count = 0
     resigned_count = 0
-    
+
     # Iterate over all sessions
     for session in Session.objects.all():
         try:
             session_data = session.get_decoded()
         except Exception:
             continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)